src/taipy/core/_entity/_migrate.py

```diff
@@ -295,10 +304,10 @@ def __connect_mongodb(db_host: str, db_port: int, db_username: str, db_password:
 
 
 def _load_all_entities_from_mongo(
-    hostname: str = "localhost",
-    port: int = 27017,
-    user: str = "taipy",
-    password: str = "password",
+    hostname: str,
+    port: int,
+    user: str,
+    password: str,
```

Comment:
We should have as a default value the default value of the configuration.
What do you think?

Reply:
The default value should be provided on the `_migrate_mongo_entities()` method. I removed these from all other methods to prevent any conflict in values in the future.